### DISCUSSION TITLE
v0.dev cannot use it anymore!

### DISCUSSION THREAD

**Discussion by janlauber** [vercel/community#4722] [Account Issue]
### Question

Hi there

I subscribed for 50 bucks to your v0.dev closed beta but now I cannot use it anymore!
I still have like 11k credits left but when i try to use your service it shows the following message: You are not authorized to perform this action.

Can you please help me fix this?

Thanks,
Jan

**Comment by jaredpalmer** [comment]
Thanks for the message. We are investigating now.

**Reply by janlauber** [reply]
Thanks, I couldn't use it now for a day...

**Reply by janlauber** [reply]
@jaredpalmer can you please fix my issue? it seems like it takes forever and I've paid for the whole month to use it...
